persistence/sql/sqlsaga_required-in-some-versions_sqlpersistence_[4,5).partial.md

```diff
@@ -0,0 +1 @@
+INFO: In order to use the normal `Saga<T>` class for sagas, upgrade to at least SQL Persistence 4.2.
```

Comment:
> the normal `Saga<T>` class

the NServiceBus Core `Saga<T>` class or the NServiceBus Core `Saga<T>` base class

Reply:
I made this change. I don't know why GitHub is still showing it this way!